xml/art_migration.xml

```diff
@@ -0,0 +1,692 @@
+    %entities;
+]>
+<article version="5.1" xml:lang="en" xml:id="art.hos.migration"
+  xmlns="http://docbook.org/ns/docbook"
+  xmlns:xi="http://www.w3.org/2001/XInclude"
+  xmlns:xlink="http://www.w3.org/1999/xlink">
+ <info>
+  <title>Migration Guide</title>
+  <date><?dbtimestamp format="B d, Y"?></date>
+  <!-- <xi:include href="common_copyright_gfdl.xml"/> -->
+  <!--<xi:include href="authors.xml"/>-->
+ </info>
+ <para>
+  <remark role="clarity">taroth 2018-02-27: who is the target audience for this document?
+  cloud operators? cloud administrators?</remark>
+  <remark>taroth 2018-02-27: in terms of terminology, I guess that 'seed', 'deployer'
+  should be changed to 'seed nodes', 'deployer nodes' etc.</remark>
+ </para>
+<sect1>
+  <title>Migration from HOS 5.x to HOS 5.SCP</title>
+  <para>
+    The migration procedure is based on the following assumptions:
+  </para>
+  <itemizedlist>
+    <listitem>
+      <para>
+	The customer is responsible for backing up all the control plane and deployer nodes to an external
+data store before starting the migration.
+      </para>
+      <itemizedlist>
+	<listitem>
+	  <para>
+	    We do not support rollback of migration after the process has been started.
+	   </para>
+	  </listitem>
+      </itemizedlist>
+    </listitem>
+    <listitem>
+      <para>
+	Metering, monitoring, and logging (MML) data will not be migrated (a
+	clean installation will be done using a new MML back-end included with HOS 5.SCP).
+    </para>
+    </listitem>
+    <listitem>
+      <para>
+	Any data that is not on a disk separate from the primary root partition
+	will be lost during re-imaging. This includes &swift; objects and audit logs.
+    </para>
+    </listitem>
+    <listitem>
+      <para>
+	All &compnode;s are running supported Linux distributions:
+    </para>
+    <itemizedlist>
+      <listitem>
+	<para>
+	  &sls; 12 SP3 for SLES &compnode;s
+	 </para>
+	</listitem>
+	<listitem>
+	  <para>
+	    RHEL 7.3 for RHEL &compnode;s
+	   </para>
+	  </listitem>
+    </itemizedlist>
+    </listitem>
+    <listitem>
+      <para>
+	There are only supported input model services.
+    </para>
+    <itemizedlist>
+      <listitem>
+	<para>
+	There is no embedded HOS Ceph or VSA.
+	 </para>
+	</listitem>
+    </itemizedlist>
+    </listitem>
+    <listitem>
+      <para>
+	Only supported third-party integrations and extensions are installed.
+    </para>
+    <itemizedlist>
+      <listitem>
+	<para>
+	  There are no unsupported customized virtual environments.
+	 </para>
+	</listitem>
+    </itemizedlist>
+    </listitem>
+  </itemizedlist>
+  <procedure>
+    <title>Preparation Phase</title>
+<step>
+  <para>
+    Run validation playbook to ensure that the current cloud load and configuration
+    is compatible with the migration process:
+   <remark role="clarity">taroth 2018-02-27: what is 'validation playbook'? the name of a script?
+    then it should be tagged with &lt;literal/&gt;</remark>
```

Comment:
The actual file name is `pre-migration-validations.yml`, so I think the way it is mentioned basically fine and does not need a literal imo. However, it might be debatable whether you should just mention the file name like: `Run the playbook <filename>pre-migration-validations.yml</filename>...`